upgrade/UPGRADE-v9.1.1-dev.md

```diff
@@ -40,6 +40,9 @@ There you can find links to upgrade notes for other versions too.
   - if you extended `Advert` type, you can remove duplicate definitions in `AdvertCode` and `AdvertImage` types
   - if you extended `Price` type, you can remove duplicate definitions in `ProductPrice` type
 
-- deprecated method `Shopsys\FrameworkBundle\Component\Elasticsearch\Exception\ElasticsearchIndexException::indexAlreadyExists()` ([#2233](https://github.com/shopsys/shopsys/pull/2233))
+- deprecated methods for throwing exceptions  ([#2233](https://github.com/shopsys/shopsys/pull/2233))
```

Comment:
We want upgrade notes to have directing approach, so you should update text to "replace deprecated methods with exceptions"

Reply:
This is not relevant any more. The upgrade note has changed when reverting introducing unused exception.